modules/nf-core/samtools/view/main.nf

```diff
@@ -11,6 +11,7 @@ process SAMTOOLS_VIEW {
     tuple val(meta), path(input), path(index)
     tuple val(meta2), path(fasta)
     path qname
+    val index_format
```

Comment:
note for me to check: why can't this be inferred from the input

Reply:
https://github.com/nf-core/modules/pull/7481